requirements.txt

```diff
@@ -1,7 +0,0 @@
-affine>=1.3.0
-boto3>=1.2.4
-cligj
-enum34
-numpy>=1.8
-snuggs>=1.2
-setuptools
```

Comment:
@perrygeo That was my fear.  I don't know that much about distributing non-standard packages, but can we just parse the requirement files in `setup.py` instead as long as we aren't using any `pip` flags?  Totally willing to revert if not.